test/unit/lib/email-service-spec.js

```diff
@@ -66,6 +66,22 @@ describe('Email Service', () => {
       EmailService.prototype._includeDate.should.not.have.been.called;
     });
 
+    it('sets subject to email-service instance if passed object', () => {
+      const subject = {
+        customer: '',
+        caseworker: ''
+      };
+      emailService = new EmailService({data, subject});
+      emailService.subject.should.be.equal(subject);
```

Comment:
yes, deep equal checks equality, as `{} !== {}` but `{}.deep.equals({})`, in this case it is the exactly the same object so `var obj = {}; obj === obj`